eng/Signing.props

```diff
@@ -96,6 +96,12 @@
     <_DotNetFilesToExclude Include="$(BaseRedistNetCorePath)win-arm\host\**\*.dll" CertificateName="None" />
     <_DotNetFilesToExclude Include="$(RedistNetCorePath)dotnet.exe" CertificateName="None" />
     <FileSignInfo Include="@(_DotNetFilesToExclude->'%(FileName)%(Extension)'->Distinct())" CertificateName="None" />
+
+    <!--
+      We include the Microsoft.Data.SqlClient.dll assembly in our global tool 'dotnet-sql-cache'.
+      It is already signed by that team, so we don't need to sign it.
+    -->
+    <FileSignInfo Include="Microsoft.Data.SqlClient.dll" CertificateName="None" />
```

Comment:
Do we still validate that the assembly **is** signed, even when we label it as not to be signed by us? Since this DLL comes from a different team, they handle the signing, but I would like us to be validating that it's all signed (in case they miss signing sometime).

For additional context: We're using the OOB SqlClient in `dotnet-sql-client` which has to embed all it's dependencies (since it's a global tool). Hence this DLL now appears in the dotnet-sql-client nupkg. We don't ship this tool in the SDK nor in any shared framework.

Reply:
cc @aspnet/build

Reply:
As far as I can tell, we do _not_ do signing validation in this repo at the moment. That'll be corrected as I complete aspnet/AspNetCore#11924 because the new stages include signing validation.

FYI everywhere else, we use the generic release pipeline (https://dnceng.visualstudio.com/internal/_release?view=mine&definitionId=39 aka "Public Dev. Release") which does signing validation for everything it publishes.

Reply:
Doing validation on publish is a reasonable stop-gap until #11924 goes in, thanks!

Reply:
I don't anticipate a problem, just being paranoid ;)